emukit/core/loop/outer_loop.py

```diff
@@ -54,18 +54,22 @@ def __init__(self, candidate_point_calculator: CandidatePointCalculator,
         self.loop_start_event = EventHandler()
         self.iteration_end_event = EventHandler()
 
-    def run_loop(self, user_function: Union[UserFunction, Callable], stopping_condition: Union[StoppingCondition, int],
-                 context: dict=None) -> None:
+    def run_loop(self, user_function: Union[UserFunction, Callable],
+                 stopping_condition: Union[StoppingCondition, int, List[StoppingCondition]],
+                 context: dict = None) -> None:
         """
         :param user_function: The function that we are emulating
-        :param stopping_condition: If integer - a number of iterations to run, if object - a stopping condition object
-                                   that decides whether we should stop collecting more points
+        :param stopping_condition: If integer - a number of iterations to run, if object (or list of such objects) - a stopping
+                        condition object that decides whether we should stop collecting more points
```

Comment:
This comment should explain how the list is handled (the fact that we use "or" and not say "and"). Don't be afraid to make it a huge multi-line thing